content/webapp/services/prismic/transformers/articles.ts

```diff
@@ -104,6 +108,22 @@ export function transformArticle(document: ArticlePrismicDocument): Article {
     ? (transformLabelType(data.format) as Format<ArticleFormatId>)
     : undefined;
 
+  // Calculating the full reading time of the article by getting all article text
+  function allArticleText(genericBody) {
+    return genericBody
+      .filter(genericBody => genericBody.type === 'text')
+      .map(element => element.value)
+      .flat()
+      .map(element => element.text)
+      .join('');
+  }
+  let readingTimeInMinutes;
+  if (format?.title === 'Article') {
+    readingTimeInMinutes = readingTime(allArticleText(genericFields.body)).text;
+  } else {
+    readingTimeInMinutes = null;
+  }
```

Comment:
couple of nits:

* Would it be worth pulling this into a standalone function, and providing examples of output? I haven't used the `readingTime` library – is it what adds the unit (e.g. "3 min")
* Ternary operator could make L120–5 a bit more concise
* Do we want to `.join('')` or `.join(' ')`? e.g. if you have two bits of text `Red Robot` and `Blue Box`, I think `allArticleText()` would get them as `Red RobotBlue Box`, which is 3 words instead of 4.

Reply:
oh, and one more nit reviewing the screenshot (sorry!):

> reading time **6 min read**

feels redundant, if we could simplify it to

> reading time **6 min**

I think that's a little nicer.

Reply:
Thanks for the thorough review, i'm working my way through all of the suggestions, but overall agree and especially grateful for the last comment on reading time wording 👍